tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -338,6 +373,7 @@ private void assertExitCodeIsOne(String... args) {
     }
 
     private List<Row> expectedOffsetsWithInternal() {
+
```

Comment:
could you please revert this unnecessary change?

Reply:
Sure & reverted